rest-model/src/main/java/com/gentics/mesh/parameter/ImageManipulationParameters.java

```diff
@@ -26,67 +26,96 @@
 	public static final String RECT_QUERY_PARAM_KEY = "rect";
 
 	public static final String CROP_MODE_QUERY_PARAM_KEY = "crop";
-	
+
 	public static final String RESIZE_MODE_QUERY_PARAM_KEY = "resize";
 
 	public static final String FOCAL_POINT_DEBUG_PARAM_KEY = "fpdebug";
 
+	public static final String AUTO = "auto";
+
 	/**
 	 * Return the image width.
-	 * 
+	 *
 	 * @return
 	 */
-	default Integer getWidth() {
-		return toInteger(getParameter(WIDTH_QUERY_PARAM_KEY), null);
+	default String getWidth() {
+		return getParameter(WIDTH_QUERY_PARAM_KEY);
 	}
 
 	/**
 	 * Set the image width.
-	 * 
+	 *
 	 * @param width
 	 * @return Fluent API
 	 */
+	default ImageManipulationParameters setWidth(String width) {
+		setParameter(WIDTH_QUERY_PARAM_KEY, width);
+		return this;
+	}
+
+	/**
+	 * Set the image width.
+	 *
+	 * @param width
+	 * @return Fluent API
+	 * @deprecated Use {@link #setWidth(String)} instead.
+	 */
+	@Deprecated
 	default ImageManipulationParameters setWidth(Integer width) {
```

Comment:
I think both methods are valid. It is bad to suggest a dev to transform his height values to string. Let us do this for him. Thus keep the default method and don't deprecate it.